migrations/versions/a90a9f8f5398_adds_planet_model.py

```diff
@@ -0,0 +1,34 @@
+"""adds Planet Model
+
+Revision ID: a90a9f8f5398
+Revises: 
+Create Date: 2024-10-25 14:45:51.663314
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a90a9f8f5398'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('planet',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('description', sa.String(), nullable=False),
+    sa.Column('size', sa.String(), nullable=False),
```

Comment:
👀 It looks like you may be missing a migration, since your model has a different name/type for this column. Is there a migration that didn't get committed?